src/main/java/com/socrata/api/Soda2Producer.java

```diff
@@ -76,6 +78,7 @@ public static final Soda2Producer newProducerWithRequestId(final String url, Str
     public Soda2Producer(HttpLowLevel httpLowLevel)
     {
         super(httpLowLevel);
+        factory = new JsonFactory();
```

Comment:
still having trouble with these tests. For example, the ExamplesTest#readmeProducerExamples uses a hardcoded username and password and the test fails with "Too many login attempts for that login. Account temporarily disabled."